addons/binding/org.openhab.binding.squeezebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,243 @@
+		<description>Add another player to your device for synchronized playback (other player mac address)</description>
+	</channel-type>
+	<channel-type id="playListIndex">
+		<item-type>Number</item-type>
+		<label>Playlist Index</label>
+		<description>Playlist index</description>
+	</channel-type>
+	<channel-type id="currentPlayingTime">
+		<item-type>Number</item-type>
+		<label>Current Playing Time</label>
+		<description>Current Playing Time</description>
+	</channel-type>
+	<channel-type id="currentPlaylistShuffle">
+		<item-type>Number</item-type>
+		<label>Shuffle Mode</label>
+		<description>Current playlist shuffle mode</description>
+				<state>
+			<options>
+				<option value="0">No Shuffle</option>
+				<option value="1">Shuffle Songs in PLaylist</option>
+				<option value="2">Shuffle Albums in PLaylist</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="currentPlaylistRepeat">
+		<item-type>Number</item-type>
+		<label>Repeat Mode</label>
+		<description>Current playlist repeat Mode</description>
+		<state>
+			<options>
+				<option value="0">No Repeat</option>
+				<option value="1">Repeat Song</option>
+				<option value="2">Repeat Playlist</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<!-- Squeezebox variables -->
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>Title of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="remotetitle">
+		<item-type>String</item-type>
+		<label>Remote Title</label>
+		<description>Remote Title (Radio) of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="album">
+		<item-type>String</item-type>
+		<label>Album</label>
+		<description>Album name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="artist">
+		<item-type>String</item-type>
+		<label>Artist</label>
+		<description>Artist name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="year">
+		<item-type>String</item-type>
+		<label>Year</label>
+		<description>Release year of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="genre">
+		<item-type>String</item-type>
+		<label>Genre</label>
+		<description>Genre name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="coverarturl">
+		<item-type>String</item-type>
+		<label>Cover Art</label>
+		<description>Address to cover art of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="coverartdata">
+		<item-type>Image</item-type>
+		<label>Cover Art</label>
+		<description>Image data of cover art of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ircode">
```

Comment:
well, ok, let's keep it for compat reasons. But I would actually prefer, if this channel would send the IR codes directly as commands and not handling it as a "state". This should in future make it easier to integrate with the rule engine (using this channel as a trigger)